docs/policy.md

```diff
@@ -0,0 +1,169 @@
+
+`entropoetry` can also `require('assert')` - a built-in module of Node.js.
+
+
+### globals
+ All platform APIs and global variables accessible by the dependency. In this example,`entropoetry` has access to `console` and it can write to process.exitCode. Global access is read-only, unless defined by the `write` value.
+
+Write access is represented by a string to make it stand out when inspecting the policy file.
+
+## Limiting access to fields of powerful objects
+
+You may have noticed the dot notation in keys within globals, packages and builtins. It can be used to limit the reference the package gets to just a subset of fields on the object.
+
+`"Buffer.from": true` means the `Buffer` global reference will exist in the package's scope but will only contain the `from` field and no other fields.  
+Same goes for `"buffer.Buffer": true` in builtins - when the package does `const buffer = require('buffer')` it will get an object with just one reference to `Buffer` constructor.
+
+LavaMoat can often generate precise lists of fields used by the package. You can narrow them down further if need be by creating a `policy-override.json` file next to the `policy.json`.
+
+# Using policy-override to expand or limit access
+
+## Adding access where LavaMoat's policy was too strict
+
+Policy files are generated by statically analyzing the source of packages and finding out which powerful references they seem to be using. It's impossible to detect it 100% correctly because there are JavaScript constructs that can't be detected without running the code.  
+If such missing permissions are identified, it's best to doublecheck they're expected to be there and then add them to the policy-override file.
+
+Example:  
+Allow `entropoetry` access to the global `URL` class and expand access to the entire `buffer` builtin on top of the generated policy above.
+
+`./lavamoat/node/policy-override.json`:
+```
+{
+  "resources": {
+    "some-package>entropoetry": {
+      "globals": {
+        "URL": true,
+      }
+    },
+    "some-package>entropoetry>bn.js": {
+      "builtin": {
+        "buffer": true
+      }
+    }
+  }
+}
+```
+
+## Limiting access
+
+If LavaMoat's static analysis finds references used in a package, but in a functionality that's not called in your application it's reasonable to want them denied. 
+
+Example:  
+- Forbid requiring of `assert` by `entropoetry`
+- Deny access to `console` with the exception of `console.log`
+
+`./lavamoat/node/policy-override.json`:
+```
+{
+  "resources": {
+    "some-package>entropoetry": {
+      "builtin": {
+        "assert": false
+      },
+      "globals": {
+        "console": false,
+        "console.log": true
+      }
+    }
+  }
+}
+```
+
+### ⚠️ Unsupported deny-list style policy
+
+> **Warning**  
+> Explicitly allowing access to an entire item and then attempting to narrow it down is not supported. The order of items in the JSON file should not matter.
```

Comment:
Perhaps:

```
Say you have an object and you want to allow most of its properties to be accessible but deny a small subset of those properties. You might think this would work:
```